src/pages/components/contained-list/usage.mdx

```diff
@@ -69,12 +69,34 @@ actions and interactive elements.
 
 ## Live demo
 
-<InlineNotification>
-
-**Coming soon:** The live demo for contained list is underdevelopment and will
-be available soon.
-
-</InlineNotification>
+<ComponentDemo
+  components={[
+    {
+      id: 'contained-list',
+      label: 'Contained list',
+    },
+  ]}
+>
+  <ComponentVariant
+    id="contained-list"
+    knobs={{
+      ContainedList: ['kind', 'size', 'isInset'],
```

Comment:
The `ComponentDemo` doesn't seem to be changing for me when interacting with these knobs. is that part of the styles being broken due to Gatsby carbon-react version ?

Reply:
I triggered a redeploy without a cache and now it looks good https://carbondesignsystem-git-fork-andrean-e9531e-carbon-design-system.vercel.app/

Reply:
@andreancardona can we set a larger width on the container, so the component doesn't look so squished?

Reply:
oh yes, I thought it was the styling - I'll make it larger!